src/ssd1306.toit

```diff
@@ -81,7 +81,7 @@ class I2cSSD1306_ extends SSD1306:
 /**
 Black-and-white driver for an SSD1306 or SSD1309 connected over SPI.
 */
-class SpiSSD1306_ extends SSD1306:
+class SpiSsd1306_ extends SSD1306:
```

Comment:
Should extend Ssd1306, surely?

Reply:
No. We are returning these classes also from the deprecated `SSD1306` constructors. As such they still need to extend the `SSD1306`.